projects/admin/src/app/record/detail-view/document-detail-view/dialog-import/dialog-import.component.html

```diff
@@ -18,25 +18,34 @@
   <h4 class="modal-title pull-left" translate>Import</h4>
 </div>
 <div class="modal-body">
-  @if (records.length < 2) {
-    <h5 translate>A document is available</h5>
-  } @else {
-    <h5 translate>Documents are available</h5>
+  @if (records.length > 0) {
```

Comment:
Nice work! 

Not blocking but I would display the h5 anytime the modal is displayed:

```typescript
@if (records.length > 0 || warning) {
    <h5>{{ 'This document may already exist' | translate }}:</h5>
    }
@if (records.length) {
    <ul class="list-unstyled mb-0">
         @for (record of records; track record) {
```